tests/FSharp.Compiler.Service.Tests/MultiProjectAnalysisTests.fs

```diff
@@ -859,16 +851,16 @@ let ``In-memory cross-project references to projects using generative type provi
     if not (File.Exists tpDLL) then 
         failwith $"expect {tpDLL} to exist"
     let optionsTestProject = 
-        {       ProjectFileName = __SOURCE_DIRECTORY__ + @"/data/TestProject/TestProject.fsproj"
+        {       ProjectFileName = __SOURCE_DIRECTORY__ ++ @"../service/data/TestProject/TestProject.fsproj"
                 ProjectId = None
                 SourceFiles = 
-                    [|  __SOURCE_DIRECTORY__ + @"/data/TestProject/TestProject.fs" |]
+                    [|  __SOURCE_DIRECTORY__ ++ @"../service/data/TestProject/TestProject.fs" |]
                 Stamp = None
                 OtherOptions =
                  [|yield "--simpleresolution"
                    yield "--noframework"
                    yield "--out:" + __SOURCE_DIRECTORY__ + @"/../../artifacts/bin/TestProject/Debug/netstandard2.0/TestProject.dll"
-                   yield "--doc:" + __SOURCE_DIRECTORY__ + @"/data/TestProject/bin/Debug/TestProject.xml"
+                   yield "--doc:" + __SOURCE_DIRECTORY__ ++ @"../service/data/TestProject/bin/Debug/TestProject.xml"
```

Comment:
Not sure what the `++` operator is, I guess from some helper module. Anyway, if it is needed here, should it not also be used 1 line above?

Reply:
++ is just our helper alias for Path.Combine. It should be used pretty much everywhere in the codebase to avoid problems with slashes. Here I was changing the path so changed this as well, updating it in other places could be a separate chore :)